lib/simple_cov/formatter/terminal.rb

```diff
@@ -50,7 +50,7 @@ class << self
     )
 
     def setup_rspec
-      return if @rspec_is_set_up || !defined?(RSpec)
+      return if @rspec_is_set_up
```

Comment:
The `else` branch here remains uncovered, because we cannot really test it, for the reason mentioned in the comment below.